documentation/SupportingDocuments/Population Density API - Algorithm Initial Proposal.md

```diff
@@ -0,0 +1,35 @@
+<ol><li>For mobile lines with persistent records, the last cell in which each user has been connected in each interval (considering intervals of 15 minutes) will be associated.</li>
+<li>It is common to have intervals for which a user does not show activity, for
+them it will be assumed that the user remains in the same cell in which he was last seen active.</li></ol>
+<strong>Output 2</strong> &rarr; Valid UEs per cell in each time interval</li><br><br>
+<li><strong>Counting and Aggregation:</strong> A count of users is performed per cell and interval.
+On this count, a statistical analysis is performed with the elimination of outliers.
+Records associated with cells with less than K users in each time interval (k –
+anonymity) are discarded.<br>
+<strong>Output 3</strong> &rarr;  Total number of UEs connected per cell in each time interval,
+considering privacy.</li><br>
+<li><strong>Spatial indexing (no personal data is processed):</strong> the space will be divided into units (grids), to associate the grids that overlap with each cell coverage area.<br>
+<strong>Output 4</strong> &rarr; Regular grid covering the required area, over the cells coverage
+areas.</li><br>
+<li><strong>Distribution and aggregation:</strong> Taking into account the coverage area of the cell, users are going to be distributed, homogeneously among the grids (currently 150m x 150m or larger) associated with the cell, and the process is repeated for all cells to stimate the number of devices based on equal users distribution. As we are assuming that the distribution of users in the coverage area of each cell is uniform, we are introducing an error/noise in the distribution of users by time interval, which will be transferred to population density predictions that contribute to reducing the risk of user reidentification. Each grid on the map can typically be served by several cells of different technologies and frequency bands. To obtain the number of users per grid and time interval, an aggregation is performed.<br>
+<strong>Output 5</strong> &rarr; Number of UEs per grid, based on distribution in the cells covering that area</li><br>
+<li><strong>Prediction:</strong> Based on the historical information of users by grid and interval, the prediction of users by grid and interval is made for each time interval of the future. The algorithm identifies a day of the week and time frame for which the prediction is requested. Then, it will read the records associated for the same weekday and time frame for the previous 4 weeks.
+The population density forecast for next Tuesday should be similar to the one observed in the last four Tuesdays. The forecast, then, will be the average of the records of the previous 4 Tuesdays at the same time frame.<br>
+<strong>Output 6</strong> &rarr; Users per grid, considering historical and calendar data</li><br>
+<li><strong>Extrapolation:</strong> Finally, the users of the MNO mobile network are extrapolated
```

Comment:
As far as I understand step 6 predicts number of devices, and now we extrapolate number of devices (not users) to population

Reply:
As previous comment, users are obtained from step 6